### DISCUSSION TITLE
7월 2주차 회의

### DISCUSSION THREAD

**Discussion by jho2301** [woowacourse/tecoble#422] [General]
## 안건

**Comment by hybeom0720** [comment]
[근로 개선 6월팀]
이전의 태그 페이지 이동을 위한 메인 페이지 Nav-Bar를 추가해두었습니다.
하지만 태그 페이지가 취소되면서 반영을 하지 않게 되었는데요.
메인 페이지에서 기본적인 Nav-bar를 추가하는 안건에 대해서 논의해보고 싶습니다.
